charmhub/client.go

```diff
@@ -100,10 +101,11 @@ func (c Config) BasePath() (charmhubpath.Path, error) {
 
 // Client represents the client side of a charm store.
 type Client struct {
-	url           string
-	infoClient    *InfoClient
-	findClient    *FindClient
-	refreshClient *RefreshClient
+	url            string
+	infoClient     *InfoClient
```

Comment:
Considering this is in essence a client facade for different implementations, I'm unsure if it would be useful. Maybe from an API facade side, but I'm still unsure about that, as you should probably provide a client interface.